squad/frontend/templates/squad/build-nav.jinja2

```diff
@@ -4,6 +4,17 @@
     » {% include "squad/_unfinished_build.jinja2" %} <a class="h2 text-primary" href="{{build_url(build)}}">{{ _('Build %s') % build.version }}</a>
 </h2>
 
+{% if build.patch_source and build.patch_id %}
```

Comment:
I would also check if build.patch_url is not None, just in case something wrong happens with `update_build_patch_url`

Reply:
done